actions/lib/ansible_base.py

```diff
@@ -43,10 +43,13 @@ def _parse_extra_vars(self):
                 var_list = [n.strip() for n in var_list]
 
                 # Separate @file vars from kwarg vars
+                json_vars = list()
                 file_vars = list()
                 kwarg_vars = list()
```

Comment:
So, maybe instead of separate dicts, the var_list should be converted to a list of tuples (type, var)